### PR TITLE
#101 Adding files to open path and line to the ide url

### PR DESCRIPTION
**Pull request by sunix**
[Review](http://localhost:8080/f?id=factoryrprp0hoptbfqav3w)

### CODE REVIEW

File: src/main/java/io/fabric8/che/starter/model/workspace/Workspace.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
     private WorkspaceRuntime runtime;
     private WorkspaceConfig config;
     private List<WorkspaceLink> links;
+    private List<WorkspaceFileToOpen> filesToOpen;
```
Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/model/workspace/Workspace.java]: @sunix should it really be on Workspace DTO level. I expected that platform will just pass line & filePath parameters and che-starter will change WorksapceLink (ide url) before returning the Workspace object

Comment (sunix) [inline src/main/java/io/fabric8/che/starter/model/workspace/Workspace.java]: you expected a new end point ?

Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/model/workspace/Workspace.java]: not new PATCH /workspace with optional parameters (line & path)

**Comment by ibuziuk** [comment]
@aslakknutsen am I correct that platform is going to pass line & path parameters in PATCH /workspace body ?

**Comment by ibuziuk** [comment]
Putting on hold due to che-starter deprecation - https://issues.jboss.org/browse/CHE-254

**Comment by ibuziuk** [comment]
@sunix closing this one. I guess it is not going to be a part of che-starter API due to it's deprecation

**Pull request closed without merging**